chainercv/links/detection_link.py

```diff
@@ -0,0 +1,39 @@
+import chainer
+
+
+class DetectionLink(chainer.Link):
```

Comment:
Why inherit from `chainer.Link`?
I think that most of the classes that inherit from this would want to inherit from `chainer.Chain`, which can have children links.